<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="currentIndex">
       <number>7</number>
      </property>
      <widget class="QWidget" name="Login">
       <widget class="QGroupBox" name="groupBox">
        <property name="geometry">
         <rect>
          <x>220</x>
          <y>250</y>
          <width>351</width>
          <height>261</height>
         </rect>
        </property>
        <property name="title">
         <string>Log in</string>
        </property>
        <widget class="QWidget" name="layoutWidget">
         <property name="geometry">
          <rect>
           <x>130</x>
           <y>90</y>
           <width>191</width>
           <height>111</height>
          </rect>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QLineEdit" name="username"/>
          </item>
          <item>
           <widget class="QLineEdit" name="password">
            <property name="echoMode">
             <enum>QLineEdit::Password</enum>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="layoutWidget">
         <property name="geometry">
          <rect>
           <x>30</x>
           <y>100</y>
           <width>64</width>
           <height>91</height>
          </rect>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QLabel" name="usernamelabel">
            <property name="text">
             <string>Username</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="passwordlabel">
            <property name="text">
             <string>Password</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="layoutWidget">
         <property name="geometry">
          <rect>
           <x>70</x>
           <y>220</y>
           <width>251</width>
           <height>32</height>
          </rect>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QPushButton" name="clearbutton">
            <property name="text">
             <string>Clear</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="loginbutton">
            <property name="text">
             <string>Login</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
       <widget class="QLabel" name="logo">
        <property name="geometry">
         <rect>
          <x>250</x>
          <y>10</y>
          <width>341</width>
          <height>231</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="AdminPage">
       <widget class="QPushButton" name="displaycollege">
        <property name="geometry">
         <rect>
          <x>240</x>
          <y>80</y>
          <width>311</width>
          <height>32</height>
         </rect>
        </property>
        <property name="text">
         <string>Colleges Section</string>
        </property>
       </widget>
       <widget class="QPushButton" name="displaysouvenirs">
        <property name="geometry">
         <rect>
          <x>240</x>
          <y>220</y>
          <width>311</width>
          <height>32</height>
         </rect>
        </property>
        <property name="text">
         <string>Souvenirs Section</string>
        </property>
       </widget>
       <widget class="QPushButton" name="logout">
        <property name="geometry">
         <rect>
          <x>240</x>
          <y>430</y>
          <width>301</width>
          <height>32</height>
         </rect>
        </property>
        <property name="text">
         <string>Logout</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="viewcollegedistance">
       <widget class="QTableView" name="collegedisplay">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>61</y>
          <width>691</width>
          <height>261</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="backcollegedisplay">
        <property name="geometry">
         <rect>
          <x>230</x>
          <y>450</y>
          <width>291</width>
          <height>32</height>
         </rect>
        </property>
        <property name="text">
         <string>Back</string>
        </property>
       </widget>
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>180</x>
          <y>340</y>
          <width>381</width>
          <height>100</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_9">
         <item>
          <widget class="QPushButton" name="viewdistance">
           <property name="text">
            <string>View Distance from Saddleback</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="viewallcolleges">
           <property name="text">
            <string>View all colleges</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="loadcolleges">
           <property name="text">
            <string>Load Colleges</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QWidget" name="Souvenirs">
       <widget class="QComboBox" name="souvenircomboBox">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>10</y>
          <width>231</width>
          <height>32</height>
         </rect>
        </property>
       </widget>
       <widget class="QTableView" name="SouvenirView">
        <property name="geometry">
         <rect>
          <x>360</x>
          <y>10</y>
          <width>381</width>
          <height>331</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="AddSouvenirbtn">
        <property name="geometry">
         <rect>
          <x>600</x>
          <y>490</y>
          <width>112</width>
          <height>32</height>
         </rect>
        </property>
        <property name="text">
         <string>Add</string>
        </property>
       </widget>
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>490</x>
          <y>370</y>
          <width>60</width>
          <height>101</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>College</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Item</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Price</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>590</x>
          <y>370</y>
          <width>127</width>
          <height>111</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <widget class="QLineEdit" name="CollegeLine"/>
         </item>
         <item>
          <widget class="QLineEdit" name="ItemLine"/>
         </item>
         <item>
          <widget class="QLineEdit" name="PriceLine"/>
         </item>
        </layout>
       </widget>
       <widget class="QPushButton" name="Deletesouvenirbtn">
        <property name="geometry">
         <rect>
          <x>60</x>
          <y>100</y>
          <width>141</width>
          <height>32</height>
         </rect>
        </property>
        <property name="text">
         <string>Delete</string>
        </property>
       </widget>
       <widget class="QLabel" name="souveniritem">
        <property name="geometry">
         <rect>
          <x>197</x>
          <y>260</y>
          <width>121</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QPushButton" name="EditSouvenirbtn">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>420</y>
          <width>111</width>
          <height>32</height>
         </rect>
        </property>
        <property name="text">
         <string>Edit</string>
        </property>
       </widget>
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>150</x>
          <y>360</y>
          <width>127</width>
          <height>131</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <item>
          <widget class="QLabel" name="editsouvenirlbl1">
           <property name="text">
            <string>TextLabel</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="editsouvenirlbl2">
           <property name="text">
            <string>TextLabel</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="priceeditsouvenir"/>
         </item>
        </layout>
       </widget>
       <widget class="QPushButton" name="backsouvenir">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>490</y>
          <width>111</width>
          <height>32</height>
         </rect>
        </property>
        <property name="text">
         <string>Back</string>
        </property>
       </widget>
       <widget class="QPushButton" name="Loadsouvenirfile">
        <property name="geometry">
         <rect>
          <x>310</x>
          <y>480</y>
          <width>181</width>
          <height>32</height>
         </rect>
        </property>
        <property name="text">
         <string>Load Souvenir File</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="Studentcustomertrip">
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>250</x>
          <y>60</y>
          <width>291</width>
          <height>351</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_6">
         <item>
          <widget class="QPushButton" name="CustomTrip">
           <property name="text">
            <string>Custom Trip</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="ShortestTrip">
           <property name="text">
            <string>Shortest Trip</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="backcollegeBTN">
           <property name="text">
            <string>Log out</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QWidget" name="CustomTrippage">
       <widget class="QTableView" name="Customtourtable">
        <property name="geometry">
         <rect>
          <x>365</x>
          <y>40</y>
          <width>371</width>
          <height>431</height>
         </rect>
        </property>
       </widget>
       <widget class="QComboBox" name="Collegecampusdrop">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>40</y>
          <width>201</width>
          <height>32</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="Backcustomtrip">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>460</y>
          <width>201</width>
          <height>32</height>
         </rect>
        </property>
        <property name="text">
         <string>Back</string>
        </property>
       </widget>
       <widget class="QLabel" name="startcollegelbcustom">
        <property name="geometry">
         <rect>
          <x>60</x>
          <y>130</y>
          <width>261</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLabel" name="startcollegepick">
        <property name="geometry">
         <rect>
          <x>60</x>
          <y>80</y>
          <width>161</width>
          <height>41</height>
         </rect>
        </property>
        <property name="text">
         <string>Pick Your Start College</string>
        </property>
       </widget>
       <widget class="QLabel" name="numberofcollegelbl">
        <property name="geometry">
         <rect>
          <x>60</x>
          <y>200</y>
          <width>261</width>
          <height>41</height>
         </rect>
        </property>
        <property name="text">
         <string>Number of Colleges added to the tour</string>
        </property>
       </widget>
       <widget class="QLabel" name="displaynumberofcollege">
        <property name="geometry">
         <rect>
          <x>60</x>
          <y>250</y>
          <width>58</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QPushButton" name="startcustomtrip">
        <property name="geometry">
         <rect>
          <x>60</x>
          <y>290</y>
          <width>181</width>
          <height>32</height>
         </rect>
        </property>
        <property name="text">
         <string>Start Algorithm</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="ShortestTrippage">
       <widget class="QTableView" name="tableView">
        <property name="geometry">
         <rect>
          <x>415</x>
          <y>30</y>
          <width>321</width>
          <height>191</height>
         </rect>
        </property>
       </widget>
       <widget class="QComboBox" name="shortesttripcombo">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>90</y>
          <width>261</width>
          <height>32</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="pickcampuslbl">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>30</y>
          <width>191</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Pick your first campus</string>
        </property>
       </widget>
       <widget class="QLabel" name="displaynumberofcampus">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>190</y>
          <width>271</width>
          <height>41</height>
         </rect>
        </property>
        <property name="text">
         <string>How many campuses you want to visit</string>
        </property>
       </widget>
       <widget class="QSpinBox" name="spinBox">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>230</y>
          <width>231</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="numberofcampuses">
        <property name="geometry">
         <rect>
          <x>110</x>
          <y>290</y>
          <width>16</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLabel" name="firstshorttrip">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>140</y>
          <width>16</width>
          <height>16</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <strikeout>true</strikeout>
         </font>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>340</y>
          <width>231</width>
          <height>151</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_7">
         <item>
          <widget class="QPushButton" name="startshorttrip">
           <property name="text">
            <string>Start Algorithm</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="backshorttrip">
           <property name="text">
            <string>Back</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QTableWidget" name="tableWidget">
        <property name="geometry">
         <rect>
          <x>430</x>
          <y>280</y>
          <width>256</width>
          <height>192</height>
         </rect>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="Souvenirtourpage">
       <widget class="QLabel" name="Souvenirtriplbl">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>60</y>
          <width>141</width>
          <height>71</height>
         </rect>
        </property>
        <property name="text">
         <string>Pick Your Souvenirs</string>
        </property>
       </widget>
       <widget class="QPushButton" name="doneSouvenir">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>470</y>
          <width>151</width>
          <height>32</height>
         </rect>
        </property>
        <property name="text">
         <string>Done</string>
        </property>
       </widget>
       <widget class="QTableWidget" name="SouvenirTableWidget">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="geometry">
         <rect>
          <x>280</x>
          <y>-10</y>
          <width>471</width>
          <height>371</height>
         </rect>
        </property>
        <property name="sizeAdjustPolicy">
         <enum>QAbstractScrollArea::AdjustToContents</enum>
        </property>
        <property name="editTriggers">
         <set>QAbstractItemView::NoEditTriggers</set>
        </property>
       </widget>
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>120</y>
          <width>131</width>
          <height>141</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_8">
         <item>
          <widget class="QPushButton" name="CalSouv">
           <property name="text">
            <string>Calculate</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="Clearcalc">
           <property name="text">
            <string>Clear</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>360</x>
          <y>380</y>
          <width>201</width>
          <height>18</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="grandlbl">
           <property name="text">
            <string>Grand Total is:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="grandTotal">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
